src/components/TeamResources.vue

```diff
@@ -0,0 +1,129 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import type { ITeamResource } from '../types.ts'
+
+defineProps<{
+	resources: ITeamResource[]
+	/**
+	 * The URL to see the full team page
+	 */
+	teamUrl: string
+}>()
+</script>
+
+<template>
+	<div class="team-resources">
+		<ul class="team-resources__list">
+			<li
+				v-for="resource in resources.slice(0, 5)"
+				:key="resource.id"
+				class="team-resources__box"
+				:title="resource.name"
+				:style="{ '--fallback-icon': `url('${resource.fallbackIcon}')` }">
+				<a :href="resource.url" class="team-resources__link">
+					<img
+						:src="resource.iconUrl"
+						class="team-resources__icon"
+						:alt="resource.name">
+				</a>
+			</li>
+			<li
+				v-if="resources.length > 5"
+				class="team-resources__box">
+				<a :href="teamUrl" class="team-resources__link">
+					<div class="team-resources__link-more">+{{ resources.length - 5 }}</div>
+				</a>
+			</li>
+		</ul>
+	</div>
+</template>
+
+<style scoped lang="scss">
+.team-resources {
+	--resources-list-gap: calc(2 * var(--default-grid-baseline));
+	--resource-box-size: var(--default-clickable-area);
+
+	&__list {
+		display: flex;
+		align-items: center;
+		flex-wrap: wrap;
+		gap: var(--resources-list-gap);
+		list-style: none;
+	}
+
+	&__box {
+		cursor: pointer;
```

Comment:
Is this needed? I don't see any `click` event listener on `box`. The `<a>` should be enough, no?